File: 2020/thanksgiving-contest/02 Count Fees/fees.java
// Submitted by Luke Qiao

import java.io.*;
import java.util.*;

public class Solution {

    public static void main(String[] args) {
        Scanner s = new Scanner(System.in);
        int n = s.nextInt();
        
        int ans = 0;
        if (n > 0) {
            StringBuilder str = new StringBuilder(s.next());
            for (int i = 0; i < n - 2; i++) {
                if (str.substring(i, i + 3).equals("FEE")) {
                    ans++;
                }
            }
        }
        System.out.println(ans);
    }
}

File: 2020/thanksgiving-contest/02 Count Fees/fees.cpp
// My solution

#include <bits/stdc++.h>

using namespace std;

int main()
{
    int n;
    string s;
    cin >> n >> s;
    int ans = 0;
    for(int i = 0; i<n-2; i++) {
        if(s.substr(i, 3) == "FEE") {
            ans++;
        }
    }
    cout << ans << endl;
}
